src/ray/object_manager/ownership_based_object_directory.h

```diff
@@ -87,6 +89,20 @@ class OwnershipBasedObjectDirectory : public ObjectDirectory {
   /// Internal callback function used by SubscribeObjectLocations.
   void SubscriptionCallback(ObjectID object_id, WorkerID worker_id, Status status,
                             const rpc::GetObjectLocationsOwnerReply &reply);
+
+  /// Metrics
+
+  /// Number of object locations added to this object directory.
+  uint64_t metrics_num_object_locations_added_;
+
+  /// Number of object locations removed from this object directory.
+  uint64_t metrics_num_object_locations_removed_;
+
+  /// Number of object location lookups.
+  uint64_t metrics_num_object_location_lookups_;
+
+  /// Number of object location updates.
+  uint64_t metrics_num_object_location_updates_;
```

Comment:
Can we also include this in the debug string?

Reply:
These field is reset after each metric recording, should we save the location updates per second value and include that in the debug string?